README.md

```diff
@@ -20,7 +20,11 @@ git clone https://github.com/konveyor/tackle-api-tests.git
 cd tackle-api-tests
 ```
 
-### Using Virtual Environment
+### Edit the Config File
+Before running the tests, you need to edit the `config.json` file, located in the root directory of the project.
+* Modify the `cred_git_username` and `cred_git_token` which are needed to create credentials.
```

Comment:
Instead of modify we can say Provide :


Provide your github user and token  for `cred_git_username` and `cred_git_token` which are needed to create credentials.